tools/gromacs/nvt.xml

```diff
@@ -0,0 +1,166 @@
+<tool id="gmx_nvt" name="NVT." version="@VERSION@">
```

Comment:
here as well ... name and description please, so that a user can get a glimpse of what it is doing